index.html

```diff
@@ -55,6 +55,18 @@ <h2>Next event</h2>
   <a href="{{ site.url }}/join" data-emoji="👉">Join us on Slack</a>
 </p>
 
+<form class="✉️" action="https://tinyletter.com/RemotehackSpace" method="post" target="popupwindow" onsubmit="window.open('https://tinyletter.com/RemotehackSpace', 'popupwindow', 'scrollbars=yes,width=800,height=600');return true">
+  <p>
+    <label for="tlemail">Stay informed of future Remote Hack days</label>
+  </p>
+  <p>
+    <input type="text"  name="email" id="tlemail" placeholder="your@email.com" class="⌨️"/><input type="hidden" value="1" name="embed"/><input type="submit" value="Subscribe" class="🤙"/>
+  </p>
+  <p>
+    <a class="❤️" href="https://tinyletter.com" target="_blank">Powered by TinyLetter</a>
```

Comment:
Lets add a `rel="noopener"` for the old school browsers

Reply:
🤷